plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorBase.java

```diff
@@ -184,6 +187,14 @@ static boolean isBigScript(@Nullable IEditorInput editorInput) {
         return false;
     }
 
+    public boolean getEditorServicesEnabled() {
```

Comment:
It would be nicer if it has `is` instead of `get`.

Let's assume that `Services` is in the singular form and thus we don't violate the laws of English language :)

Or you can use `are`, but I haven't seen one using that prefix.